pydantic/class_validators.py

```diff
@@ -57,7 +64,10 @@ def dec(f: AnyCallable) -> classmethod:
                 raise ConfigError(f'duplicate validator function "{ref}"')
             _FUNCS.add(ref)
         f_cls = classmethod(f)
-        f_cls.__validator_config = fields, Validator(f, pre, whole, always, check_fields)  # type: ignore
+        f_cls.__validator_config = (  # type: ignore
+            fields,
+            Validator(func=f, pre=pre, whole=whole, always=always, check_fields=check_fields),
```

Comment:
Cython required use of keyword rather than positional arguments

Reply:
shouldn't be necessary after removing `@dataclass`?